package-lock.json

```diff
@@ -1903,67 +1903,6 @@
         }
       }
     },
-    "@parse/s3-files-adapter": {
```

Comment:
@zanderisrael Can you revert the package-lock.json?

Reply:
I just deleted the package-lock.json file, Im not really sure why this is being tracked in the first place... 
It should be gitignored since the package.json file is what we need, the lock file is generated per environment when npm installing...

Reply:
@zanderisrael, it is common practice to commit package-lock. It prevents unexpected behavior because it fixates dependencies to a specific version. Without committing it, every installation of Parse Server could vary in its dependencies.